src/views/domain-page/domain-page-header-info/domain-page-header-info.types.ts

```diff
@@ -0,0 +1,28 @@
+import type { DomainInfo } from '../domain-page.types';
+
+type LoadingProps = {
+  loading: true;
+};
+
+type LoadedProps = {
+  loading: false;
+  domainInfo: DomainInfo;
+  cluster: string;
+};
+
+export type Props = LoadingProps | LoadedProps;
+
+export type DomainHeaderInfoItemContentProps = {
+  domainInfo: DomainInfo;
+  cluster: string;
+};
+
+type DomainPageHeaderInfoItem = {
```

Comment:
nit: Maybe easier to explain what is the Item by adding a postfix for Props/Config